frontend/src/components/associations/medias/PreviewCard.tsx

```diff
@@ -0,0 +1,77 @@
+import Card from "react-bootstrap/Card";
+import { TaggableModel, TagList } from "../../utils/tags/TagList";
+import React from "react";
+import "./PreviewCard.css";
+
+/**
+ * Generic component to present a media in a card
+ * @param media the media to display
+ * @param showPreview show the preview
+ * @param showDescription show the preview
+ * @param showTags show the preview
+ * @param overlayInformation true to show the information on the image on hover
+ * @param onClick callback in case the card is clicked
+ * @param props additional JSX props
+ * @constructor
+ */
+export const MediaPreviewCard = (
+  {
+    media,
+    showPreview = true,
+    showDescription = true,
+    showTags = true,
+    overlayInformation = false,
+    onClick = () => {},
+    ...props
+  } /* {
+  media: Media;
+  showPreview?: boolean;
+  showDescription?: boolean;
+  showTags?: boolean;
+  onClick?: () => void;
+}*/
```

Comment:
No type finally?

Reply:
Typing ...props is a real pain

Reply:
https://github.com/typescript-cheatsheets/react-typescript-cheatsheet/blob/master/README.md#useful-react-prop-type-examples

Does that help you?